runtime/java_test.go

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (C) 2017 XLAB, Ltd.
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+package runtime
+
+import (
+	"testing"
+
+	. "github.com/mikelangelo-project/capstan/testing"
+	. "gopkg.in/check.v1"
+	"gopkg.in/yaml.v2"
+)
+
+// Hook up gocheck into the "go test" runner.
+func Test(t *testing.T) { TestingT(t) }
+
+type javaSuite struct {
+}
+
+var _ = Suite(&javaSuite{})
+
+func (*javaSuite) TestGetBootCmd(c *C) {
+	// Simulate openjdk8-zulu-compact1's meta/run.yaml being parsed.
+	cmdConfs := map[string]*CmdConfig{
+		"openjdk8-zulu-compact1": &CmdConfig{
+			RuntimeType:      Native,
+			ConfigSetDefault: "java",
+			ConfigSets: map[string]Runtime{
+				"java": nativeRuntime{
+					BootCmd: "/java.so",
+					CommonRuntime: CommonRuntime{
+						Env: map[string]string{
+							"XMS":       "512m",
```

Comment:
This requires that we keep the actual Java package in sync, right? For example, in case we change the defaults in the real `openjdk8-zulu-compact1` package, this may cause some issues - is there a way we could get this configuration based on the current `capstan-packages` version of the java package?

Reply:
I agree.